### DISCUSSION TITLE
Problems Deploying Dino Models

### DISCUSSION THREAD

**Discussion by gusmcarreira** [open-mmlab/mmdeploy#2296] [Q&A]
Hi there, I have been trying to deploy the models for which the configs are [here](https://github.com/open-mmlab/mmdetection/tree/main/configs/dino) but as I try such I get this error

`
RuntimeError: Only tuples, lists and Variables are supported as JIT inputs/outputs. Dictionaries and strings are also accepted, but their usage is not recommended. Here, received an input of unsupported type: InstanceData
07/21 12:03:25 - mmengine - ERROR - path_to_venv/lib/python3.8/site-packages/mmdeploy/apis/core/pipeline_manager.py - pop_mp_output - 80 - "mmdeploy.apis.pytorch2onnx.torch2onnx" with Call id: 0 failed. exit.
`

Does anyone know how to solve it?

Thank you in advance!

**Comment by stihuangyuan** [comment]
meet same error
